### ISSUE TITLE
[JSTC-37] Create table help output shows prefix

### ISSUE THREAD

**Issue by sanderpick** [tablelandnetwork/js-tableland-cli#387]
When creating a table, I noticed that the prefix is auto-applied from the table name in the create statement (neat!). However, the help command still shows that you can supply a prefix. I'm not sure if that still actually works. In either case, I don't think there should be two ways to do this… or if that is needed for some reason, the help command should be clear about it.

```
tableland create [schema] [prefix]        Create a new table
```

cc @carsonfarmer @joewagner 

<sub>From [SyncLinear.com](https://synclinear.com) | [JSTC-37](https://linear.app/tableland/issue/JSTC-37/create-table-help-output-shows-prefix)</sub>

**Comment by joewagner** [comment]
@sanderpick good call on this command being unclear.  I'm dumping my thoughts below...

For context, I believe we have 5 ways to run the create command:
1. provide a schema and prefix via command args
2. provide a schema via command args.  (note: table name will not have a prefix, e.g. `_31337_3`.  This feature was added by a request that may have originated in discord. Details:  https://github.com/tablelandnetwork/js-tableland-cli/issues/306)
3. provide a full create query.  (note: this can be any number of create statements, i.e. you can create all your tables with one command)
4. call the command with no arguments, which opens a stdin prompt allowing you to freely type.  (note: I just tried this out and I'm not sure this works right. If we want to keep this feature I can open an issue to fix)
5. use the command to run a file that contains SQL.  

The full text of  `npx tableland create --help` tries to explain to explain all of this, but maybe still confusing.

Ideas:
- Reduce the number of ways to use `create`.
- Replace the current usage with a different single usage.
- Make changes to the help messages.
- other?
cc: @dtbuchholz @awmuncy

**Comment by carsonfarmer** [comment]
I think input from stdin/files is still useful. We *should* just drop support for extra flags etc though (IMO). Because there should really just be one way to do it, using a create statement. Stdin/files are also just doing it the same way (via a create statement), they are just alternative input sources.

**Comment by carsonfarmer** [comment]
I think the stdin option requires you to pipe in the content? But not 100% sure anymore?

**Comment by joewagner** [comment]
> I think the stdin option requires you to pipe in the content? But not 100% sure anymore?

That makes sense.  You _can_ just type, but the prompt never closes and only works for the first statement.

**Comment by carsonfarmer** [comment]
Does Ctrl+D properly close it out?

**Comment by joewagner** [comment]
> Does Ctrl+D properly close it out?

Yeah, which is all good, it's just a little confusing since the second (and third etc) command don't do anything.

**Comment by dtbuchholz** [comment]
I think that should be fine...I should be able to pass an empty prefix within a full create statement anyways like `create table "" ...`

**Comment by sanderpick** [comment]
Ah, makes sense! I didn't even realize that i had supplied a full create statement. That is a nice pattern. The other way that makes sense to me is with the `shell` command => just type SQL as normal, including create statements. That's possible, right? 

I don't have a strong pref on removing options here... but we should document them. If I do `tableland create --help` I just see:
```
tableland create -h
tableland create [schema] [prefix]

Create a new table

Positionals:
  schema  SQL table schema, or full create statement (skip to read from stdin)
                                                                        [string]

Options:
  -h, --help                 Show help                                 [boolean]
  -V, --version              Show version number                       [boolean]
      --baseUrl              The URL of your Tableland validator        [string]
  -c, --chain                The EVM chain to target                    [string]
      --enableEnsExperiment  Enable ENS experiment                     [boolean]
      --ensProviderUrl       Enable ENS experiment                      [string]
  -k, --privateKey           Private key string                         [string]
  -p, --providerUrl          JSON RPC API provider URL. (e.g., https://eth-rinke
                             by.alchemyapi.io/v2/123abc123a...)         [string]
      --prefix               Table name prefix (ignored if full create statement
                              is provided)                              [string]
      --ns                   ENS namespace to resolve schema from (experimental)
                                                                        [string]
  -f, --file                 Get statement from input file
``` 

I think some examples of the different patterns would be nice in the help output.
